dowhy/causal_model.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, data, treatment, outcome, graph=None,
                  estimand_type="nonparametric-ate",
                  proceed_when_unidentifiable=False,
                  missing_nodes_as_confounders=False,
+                 identify_vars=False,
```

Comment:
this is a great idea. Can you add this to the doc string for  _init_? Also make sure to mention in the docstring that identify_vars should be set to False when user is providing `common_causes` or instruments on their own. 
(otherwise identify_vars code can override the user provided ones, and also it does not make sense if no graph is given)